docs/Getting Started/Understanding Permissions/index.md

```diff
@@ -132,17 +132,50 @@ const isParameterAllowed = sdlManager.getPermissionManager().isPermissionParamet
 !@
 
 ### Checking Current Permissions of a Group of RPCs
+You can also retrieve the status of a group of RPCs. First, you can retrieve the permission status of the group of RPCs as a whole: whether or not those RPCs are all allowed, all disallowed, or some are allowed and some are disallowed. This may allow you to know, for example, if a feature you need is allowed based on the status of all the RPCs of the features.
```

Comment:
```suggestion
You can also retrieve the status of a group of RPCs. First, you can retrieve the permission status of the group of RPCs as a whole: whether or not those RPCs are all allowed, all disallowed, or some are allowed and some are disallowed. This will allow you to know, for example, if a feature you need is allowed based on the status of all the RPCs needed for the feature.
```